### DISCUSSION TITLE
Atom Files for Repositories Returning 404 - Recent Issue

### DISCUSSION THREAD

**Discussion by own3mall** [community/community#56822] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

We used to parse the atom files of our github repositories to get a list of commit changes, but it seems this isn't available anymore?  Is there any reason why? 

For example, this URL used to resolve:

https://github.com/OpenGamePanel/OGP-Website/commits/OGP-Website.atom

But it doesn't anymore.  Is there a way to re-activate this old functionality?

**Comment by own3mall** [comment]
Never mind.  The URL is supposed to be https://github.com/OpenGamePanel/OGP-Website/commits/master.atom

I forgot how this functionality worked and just trusted the bug report a user filed which was invalid.  Please disregard.

**Reply by Suthiwat1** [reply]
J

**Comment by Suthiwat1** [comment]
K
